### ISSUE TITLE
Preserve symmetry with operations

### ISSUE THREAD

**Issue by bleyerj** [bleyerj/jaxmat#16]
Operations like `jnp.sum(tensors,axis=0)` where tensors is a batched version of SymmetricTensor2 produces a Tensor2. Other operations do the same. issue is that initialized types (`SymmetricTensor2`) then do not match anymore.
Generally, we should better handle views and array operations on tensors

**Comment by bleyerj** [comment]
Closing for now.
